WordPress/gutenberg
# Keep framer-motion from updating minor version

## What?


Fixes #51893.

This is a quick patch so that the npm package updates don't fail in core. See https://github.com/WordPress/wordpress-develop/pull/4699.

## Testing Instructions





1. Delete`node_modules` and `package-lock.json`;
2. Run `npm i` and `npm run dev`;
3. Check that popovers don't crash any of the editors.

### Testing Instructions for Keyboard


## Screenshots or screencast
